src/api/app/models/event/base.rb

```diff
@@ -273,6 +275,12 @@ def _roles(role, project, package = nil)
       obj_roles(p, role)
     end
 
+    def send_to_bus
+      RabbitmqBus.publish(self.class.message_bus_queue, read_attribute(:payload))
+    rescue Bunny::Exception => e
+      logger.error "Publishing to RabbitMQ failed: #{e.message}"
+    end
+
```

Comment:
Would be nice to add a method like this to `Event::Base` in case somebody adds `after_commit :send_to_bus` without defining the message queue:
```ruby
def self.message_bus_queue
  raise NotImplementedError
end
```

Reply:
What is the difference to `NoMethodError`?

Reply:
`NotImplementedError` is more specific and as I've seen it elsewhere, commonly used as a way to indicate that this methods needs to be implemented by the sub class. Since ruby doesn't have any way of doing abstract classes like java.